data-science-onramp/data-ingestion/README.md

```diff
@@ -0,0 +1,7 @@
+# Data Ingestion
+This is part of a larger tutorial found on cloud.google.com
```

Comment:
Super annoying and tiny nit to add a period to the end of this but otherwise lgtm